packages/ckeditor5-dev-changelog/tests/utils/getchangesetfilepaths.test.ts

```diff
@@ -0,0 +1,187 @@
+/**
+ * @license Copyright (c) 2003-2025, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+import { glob } from 'glob';
+import upath from 'upath';
+import { vi, describe, it, expect, beforeEach } from 'vitest';
+import type { RepositoryConfig } from '../../src/types.js';
+import { getChangesetFilePaths } from '../../src/utils/getchangesetfilepaths.js';
+import { getRepositoryUrl } from '../../src/utils/external/getrepositoryurl.js';
+
+vi.mock( 'upath' );
+vi.mock( 'glob' );
+vi.mock( '../../src/utils/external/getrepositoryurl.js' );
+
+describe( 'getChangesetFilePaths', () => {
+	beforeEach( () => {
+		vi.mocked( upath.join ).mockImplementation( ( ...paths ) => paths.join( '/' ) );
```

Comment:
From my point of view it is not needed to mock `upath`. Let's treat this as an external library, like `lodash`.

Reply:
Fixed in [d817ddb](https://github.com/ckeditor/ckeditor5-dev/pull/1090/commits/d817ddbc815de2fff02a421dbb244018793b225a).